echopype/echodata/combine.py

```diff
@@ -105,10 +105,10 @@ def check_and_correct_reversed_time(
 
 
 def assemble_combined_provenance(input_paths):
-    return xr.Dataset(
-        data_vars=source_files_vars(input_paths),
-        attrs=echopype_prov_attrs(process_type="conversion"),
-    )
+    prov_dict = echopype_prov_attrs(process_type="combination")
```

Comment:
I see here that you changed `process_type` to `"combination"`. I agree that the name should be changed. However, maybe it is a better idea to create an issue for this (so that I can account for it in my combine routine) and keep `"conversion"` in this PR?

Reply:
Yeah, I made that change because I happened to run into that `process_type` string while fixing the combine problem, and I had not noticed that usage before. It's not inherent to this PR, so it's not a problem undoing it for the time being.

All tests **are** now running fine, though (see the CI). So I'll leave the PR as is at this time. If you still think it's best not to include it in the PR, please go ahead and create the new issue and make the change to the two instances in this PR.

Reply:
Ok, as long as all tests are passing this change is fine.